src/components/homepage/homepagetemplate.js

```diff
@@ -1,38 +1,59 @@
 import React from 'react';
 import Helmet from 'react-helmet';
-import banner from './banner.svg';
 import PropTypes from 'prop-types';
 
+import Wrenches from './wrenches';
+
 import { withStyles } from '@material-ui/core/styles';
 
 const StyleHeader = () =>
 	<Helmet>
-		<link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet"/>
+		<link href="https://fonts.googleapis.com/css?family=Poppins:400,600,700" rel="stylesheet"/>
```

Comment:
@jodymlin You should check out the `HeadFooter` component. That component will be included in every page so it will be safe to put all font dependencies here. Now, you can safely remove this font dependency and move it to `HeadFooter.js`!